### DISCUSSION TITLE
Dependency dashboard does not link to Config Migration MR

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#32077] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

38.128.6

### Please tell us more about your question or problem

On my GitLab instance, the Dependency Dashboard does mention the Configuration Migration MR, but it's not clickable. On GL, MRs are referenced using `!42`, not `#42`.

![image](https://github.com/user-attachments/assets/3fd7aa7e-1dd1-48ec-b756-cfcfadad13da)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Could you give some of the other raw markdown from the dashboard, e.g. for an open regular PR? do we link to `!123` or perhaps to a full URL? I'm not sure why they'd work while this one doesn't

**Reply by astellingwerf** [reply]
```
## Config Migration Needed

<!-- config-migration-pr-info --> See Config Migration MR: #93.

This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

...

## Open

These updates have all been created already. Click a checkbox below to force a retry/rebase of any.

 - [ ] <!-- rebase-branch=renovate/alpine-3.20-dependencies -->[Update 🏔 Alpine 3.20 dependencies](!89) (`alpine_3_20/apk-tools`, `alpine_3_20/libcrypto3`, `alpine_3_20/libssl3`, `artifactory.otxlab.net/docker-hardened/alpine`)
 - [ ] <!-- rebase-branch=renovate/quarterly-version-with-separator-0-6-weeks-ahead-of-time-2404.x -->[Update quarterly version with separator 0 6 weeks ahead of time to v2404](!94)
```

**Reply by rarkins** [reply]
I wonder if that massaging is done by this line alone? https://github.com/renovatebot/renovate/blob/c7d66d15af867623f6aee42c2a198085bfe59a16/lib/modules/platform/gitlab/index.ts#L908

**Reply by astellingwerf** [reply]
Well, that notation isn't used. Unlike getListItem that outputs `[${branch.prTitle!}](../pull/${branch.prNo})`, for the reference to the Config Migration PR, it's just `See Config Migration PR: #${configMigrationRes.prNumber}`.

**Reply by rarkins** [reply]
So it should work with a small change of syntax?

**Reply by astellingwerf** [reply]
Yes, if we'd use the Markdown notation for a link to `../pull/${configMigrationRes.prNumber}`, I'd imagine the massaging would update the link correctly.

**Reply by astellingwerf** [reply]
@rarkins, can you turn this into an issue?

**Reply by HonkingGoose** [reply, marked as answer]
See:

- #32178
